tests/test_model.py

```diff
@@ -977,45 +977,6 @@ def test_add_line_load_to_3_edges(self, expected_geometry_line_load: Geometry,
 
         TestUtils.assert_almost_equal_geometries(expected_geometry, generated_geometry)
 
-    def test_add_moving_point_load(self, expected_geometry_line_load: Geometry,
```

Comment:
why did you remove this test? It should still work

Reply:
Honestly, I overlooked it!